src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -180,6 +180,29 @@ public void VerifyRuntimeNameOnNetCoreApp()
             Assert.Same(RuntimeInformation.FrameworkDescription, RuntimeInformation.FrameworkDescription);
         }
 
+        [Fact]
+        public void VerifyFrameworkDescriptionContainsCorrectVersion()
+        {
+            var frameworkDescription = RuntimeInformation.FrameworkDescription;
+            var version = frameworkDescription.Substring(".NET".Length).Trim(); // remove ".NET" prefix
+
+            if (String.IsNullOrEmpty(version))
```

Comment:
we prefer lowercase string:

> IDE0049: “Use language keywords instead of framework type names for type references”

```suggestion
            if (string.IsNullOrEmpty(version))
```